sessionctx/variable/sysvar.go

```diff
@@ -1769,6 +1778,8 @@ const (
 	PluginDir = "plugin_dir"
 	// PluginLoad is the name of 'plugin_load' system variable.
 	PluginLoad = "plugin_load"
+	// EnableDDL indicates whether the tidb-server runs DDL statements,
+	EnableDDL = "enable_ddl"
```

Comment:
Can we name it `TiDBEnableDDL`? For `PluginDir`: this variable is actually a MySQL compatible variable, but for `EnableDDL`, MySQL has no such notion of this.